README.md

```diff
@@ -124,6 +124,95 @@ The console program `qt3piezo` comes installed via the 'nipiezojenapy' package,
 
 Similarly, this applications can be configured via command line options to match the haredware setup.
 
+# Development
+
+If you wish you make changes to qt3-utils (and hopefully merge those improvements into this repository), here are some brief instructions to get started. These instructions assume you are a member of the QT3 development team and have permission to push branches to this repo. If you are not, you can instead fork this repo into your own GitHub account, perform development and then issue a pull-request from your forked repo to this repo through GitHub. Alternatively, reach out to a maintainer of this repo to be added as a developer. 
+
+1. Create a new development python environment (using Conda, venv or virtualenv) with Python = 3.9 or greater
+
+```
+> conda create --name qt3utilsdev python=3.9
+```
+
+2. Activate that environment
+
+```
+> conda activate qt3utilsdev
+```
+
+3. Clone this repository
+
+```
+> git clone https://github.com/qt3uw/qt3-utils.git
+```
+
+4. Install qt3-utils in "editor" mode
+
+```
+> cd qt3-utils
+> pip install -e . 
+```
+
+5. Create a new branch for your work
+
+```
+> git checkout -b add-my-fancy-new-feature
+```
+
+6. Add your code and test, test, test!
```

Comment:
Ideally would be to test on actual hardware

Reply:
I address this in the comments below, but I'll also add this here.